payjoin-io/Cargo.toml

```diff
@@ -11,6 +11,13 @@ resolver = "2"
 version = "0.0.1"
 
 [dependencies]
+ohttp = { version = "0.5.1" }
+bhttp = { version = "0.5.1" }
+url = "2.2.2"
+tokio = { version = "1.12.0", features = ["full"] }
+ureq = "2.9.4"
```

Comment:
The idea of this crate is to encapsulate `payjoin`, so we can depend on a version of `payjoin` with relevant features directly.

Reply:
I think it should be a dev-dep in `payjoin`. That shouldn't be circular, is it?